src/dataregistry/db_basic.py

```diff
@@ -220,6 +206,20 @@ def __init__(
                 self._schema = schema
                 self._namespace = None
 
+        # Check `cretion_mode` is allowed
+        if creation_mode and schema is None:
+            raise DataRegistryException(
+                "`creation_mode` can only be flagged when passing a `schema`"
+            )
+
+        # Namespace schema must be either "working" or "production"
```

Comment:
After line 196 please add
```
   self._query_mode = "working"
   self._entry_mode = "working"

This will, for example, ensure that `Query.get_keyword_list` will work properly for sqlite dialect without the user having to specify anything extra.